### PR TITLE
Hosted customers won't know their project name

### PR DESCRIPTION
**Pull request by dnarain-astro**
Show customers how to get their project name from CloudUI, because customers running on Hosted won't know where their cluster is running.

### CODE REVIEW

**Review by kwcanuck — COMMENTED** [review]
@dnarain-astro can you confirm that the Google Cloud project name appears in the **ID** column as documented? Isn't the Google Cloud project name located in the **Name** column:

image

If the **ID** column reference is correct, perhaps `<gcp-project-name>` should be updated to `<gcp-project-id>` in the procedure.

**Comment by dnarain-astro** [comment]
Confirmed that it does show up in the AccountID column - see the screenshot below.  And yes, your suggestion is better - calling it project-id makes more sense given that we call the column AccountID.

image

File: astro/connect-gcp.md [hunk 1]
```diff
@@ -82,8 +82,11 @@ To grant a Deployment on Astro access to external data services on GCP, such as
     Google service accounts for Astro Deployments are formatted as follows:
 
     ```text
-    astro-<deployment-namespace>@<gcp-project-name>.iam.gserviceaccount.com
+    astro-<deployment-namespace>@<gcp-account-id>.iam.gserviceaccount.com
```
Comment (jwitz) [inline astro/connect-gcp.md:85]:
```suggestion
    astro-<deployment-namespace>@<gcp-account-id>.iam.gserviceaccount.com
```

File: astro/connect-gcp.md [hunk 2]
```diff
@@ -82,8 +82,11 @@ To grant a Deployment on Astro access to external data services on GCP, such as
...
     ```
+    
+    To locate your Google Cloud account ID, in the Cloud UI click **Clusters**. The Google Cloud account ID is located in the **Account ID** column.
+
```
Comment (jwitz) [inline astro/connect-gcp.md:90]:
```suggestion
```

**Review by kwcanuck — APPROVED** [review]

**Pull request merged**